docs/src/modules/ROOT/pages/constraints-and-score/score-calculation.adoc

```diff
@@ -1371,6 +1376,36 @@ Finally, we use <<constraintStreamsMappingTuples,mapping>> to calculate the viol
 For example, if the equipment has a `capacity` of `3`, and the `maximumOverlap` of the `resource` is `5`, then `violationAmount` will be `2`.
 If the amount is `0`, then the equipment is not being used over its capacity and we can filter the tuple out.
 
+In case no temporal data is available, it's still possible to create connected ranges using the `ConstraintCollectors.toConnectedRanges(...)` collector.
+In this case the start and end of a range needs to be provided as well as a function to calculate the difference between the two.
+
+[tabs]
+====
+Java::
++
+[source,java,options="nowrap"]
+----
+.groupBy(Job::getEquipment,
+    ConstraintCollectors.toConnectedRanges(
+                          Job::getStart,
+                          Job::getEnd,
+                          (a, b) -> b - a // difference calculator
+    )
+)
+----
+Python::
++
+[source,python,options="nowrap"]
+----
+.group_by(lambda job: job.equipment,
+          ConstraintCollectors.to_connected_ranges(
+                lambda job: job.start,
+                lambda job: job.end,
+                lambda a, b: b - a) #difference calculator
```

Comment:
Inline comments should be seperated by code by at least 2 spaces, and have a space before text.
https://peps.python.org/pep-0008/#inline-comments

```suggestion
                lambda a, b: b - a)  # Difference calculator
```